src/components/editor-page/editor-page.tsx

```diff
@@ -79,10 +87,12 @@ export const EditorPage: React.FC = () => {
 
   const setRendererToScrollSource = useCallback(() => {
     scrollSource.current = ScrollSource.RENDERER
+    console.debug('[EditorPage] Make renderer scroll source')
```

Comment:
Should these be still here?

Reply:
Yes, @mrdrogdrog and I discussed that and decided to leave that in and use a proper logging framework later on (see #1511).

Reply:
Won't change this here